hc/api/admin.py

```diff
@@ -171,6 +171,10 @@ class ChannelAnnotations(TypedDict):
     owner_email: str
 
 
+def channels_view_on_site(obj: Channel) -> str:
+    return reverse("hc-channels", args=[obj.project_code])
```

Comment:
It complains here that it cannot find `project_code` which makes sense. But if I use WithAnnotations it also fails because then it doesn't match ModelT

Reply:
Okay and that fix is clearly wrong :)